pdns/dnssecinfra.cc

```diff
@@ -137,7 +137,7 @@ std::string DNSCryptoKeyEngine::convertToISC() const
   storvector_t stormap = this->convertToISCVector();
   ostringstream ret;
   ret<<"Private-key-format: v1.2\n";
-  for(const stormap_t::value_type& value :  stormap) {
+  for(const storvector_t::value_type& value :  stormap) {
```

Comment:
```suggestion
  for(const storvector_t::value_type& value :  storvec) {
```

Reply:
IMO this clears up the confusion...

Reply:
Yes, agree, I think `for(const auto& value :  stormap) {` would work as well but I've to test it first